.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   build:
     parallelism: 1
     docker:
-      - image: circleci/elixir:1.10.1
+      - image: circleci/elixir:1.12.1
```

Comment:
This reminds me that maybe we can create a .tool-versions file with the following content: 
```
elixir 1.12.1-otp-24
erlang 24.0.2
```
What do you think?